### DISCUSSION TITLE
The new version of `config` adapts ResNet algorithm: 新版 config 适配 ResNet

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#334] [MMPreTrain]
# The new version of `config` adapts ResNet algorithm: 
 新版 config 适配 ResNet 

The new version of `config` adapts ResNet algorithm:
 MMEngine has implemented a new set of configuration mechanisms based on which we have adapted the configuration files.
 The relevant case study is available at [folder]([https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)](https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)) 
 For this task, the targeted directory for adaptation is 'configs/resnet/' with at least 10 configuration file adaptions required.

新版 config 适配 resnet算法 
 mmengine 实现了一套新的 config 机制，基于新的 config 机制进行配置文件的适配，[相关案例]([https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)](https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)) 
 该任务适配的 config 目录：configs/resnet/ ，至少完成 10 个 config 文件适配

## Task Information

**Repository**：MMPreTrain

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：aso538

**Claim Time**：2023年10月25日

**PR Link**：https://github.com/open-mmlab/mmpretrain/pull/1828

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by aso538** [comment]
Claim the mission

**Reply by aso538** [reply]
https://github.com/open-mmlab/mmpretrain/pull/1828
